testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/DevelocityPluginSmokeTest.groovy

```diff
@@ -35,16 +35,19 @@ import java.util.concurrent.ConcurrentHashMap
 class DevelocityPluginSmokeTest extends AbstractSmokeTest {
 
     enum CI {
+        // https://github.com/etiennestuder/teamcity-build-scan-plugin/releases
         TEAM_CITY(
-            AbstractSmokeTest.TestedVersions.teamCityGradlePluginRef,
+            "v0.36",
             "https://raw.githubusercontent.com/etiennestuder/teamcity-build-scan-plugin/%s/agent/src/main/resources/init-scripts/develocity-injection.init.gradle"
         ),
+        // https://github.com/jenkinsci/gradle-plugin/releases
         JENKINS(
-            AbstractSmokeTest.TestedVersions.jenkinsGradlePluginRef,
+            "gradle-2.15",
```

Comment:
💅 why not the latest version [2.16.1149.v711b_998b_0532](https://github.com/jenkinsci/gradle-plugin/releases/tag/2.16.1149.v711b_998b_0532)?

See https://github.com/jenkinsci/jep/blob/master/jep/305/README.adoc

Could we automate this as well?

https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases

Reply:
Accessing GH releases feels a bit cumbersome, and I thought about delegating this to DV, but yeah, I'll do that.

Reply:
At the same time, merging this as is is fine with me!